Projects/Presentation/Sources/RoutineCreation/View/RoutineCreationView.swift

```diff
@@ -0,0 +1,622 @@
+
+                let allDayButtonImage = executionTime == .allDay
+                    ? BitnagilIcon.checkedIcon
+                    : BitnagilIcon.uncheckedIcon
+                self?.allDayButton.setImage(allDayButtonImage, for: .normal)
+            }
+            .store(in: &cancellables)
+
+        viewModel.output.isRoutineValid
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] isRegisterButtonEnabled in
+                self?.registerButton.isEnabled = isRegisterButtonEnabled
+                self?.registerButton.backgroundColor = isRegisterButtonEnabled
+                    ? BitnagilColor.navy500
+                    : BitnagilColor.navy50
+            }
+            .store(in: &cancellables)
+    }
+
+    private func configureButtonActions() {
+        subroutineAdditionButton.addAction(
+            UIAction { [weak self] _ in
+                self?.viewModel.action(input: .addSubRoutine)
+            },
+            for: .touchUpInside)
+
+        detailInfoButton.addAction(
+            UIAction { [weak self] _ in
+                self?.detailInfoButton.isSelected.toggle()
+                if self?.detailInfoButton.isSelected ?? false {
+                    self?.detailToolTipView.showTooltip()
+                } else {
+                    self?.detailToolTipView.hideTooltip()
+                }
+            },
+            for: .touchUpInside)
+
+        repeatInfoButton.addAction(
+            UIAction { [weak self] _ in
+                self?.repeatInfoButton.isSelected.toggle()
+                if self?.repeatInfoButton.isSelected ?? false {
+                    self?.repeatToolTipView.showTooltip()
+                } else {
+                    self?.repeatToolTipView.hideTooltip()
+                }
+            },
+            for: .touchUpInside)
+
+        repeatDailyButton.addAction(
+            UIAction { [weak self] _ in
+                self?.viewModel.action(input: .configureRepeatType(type: .daily))
+            },
+            for: .touchUpInside)
+
+        repeatWeekButton.addAction(
+            UIAction { [weak self] _ in
+                self?.viewModel.action(input: .configureRepeatType(type: .week))
+            },
+            for: .touchUpInside)
+
+        allDayButton.addAction(
+            UIAction { [weak self] _ in
+                self?.viewModel.action(input: .configureExecution(type: .allDay))
+            },
+            for: .touchUpInside)
+    }
+
+    private func configureSubroutineStackView() {
+        subroutineStackView.axis = .vertical
+        subroutineStackView.spacing = Layout.subroutineStackViewSpacing
+
+        RoutineCreationViewModel.RoutineExample
+            .allCases
+            .dropFirst()
+            .enumerated()
+            .forEach {
+                let view = RoutineCreationInputView()
+                let placeholder = "\($0.offset + 1).ex) \($0.element.rawValue)"
```

Comment:
아싸 !!!!!! 찾았다 !!!!!!!! 요기군요 ㅋ.ㅋ !!! 여기 띄어쓰기 1번 해주는것이 어떤지 ?? 
```suggestion
                let placeholder = "\($0.offset + 1) .ex) \($0.element.rawValue)"
```

Reply:
띄어쓰기 확~!